HelloAPI-server/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿<div class="jumbotron">
+    <h1>ASP.NET</h1>
```

Comment:
Could you add something here that indicates that this is an API.

Reply:
:+1: Updated with a couple of sentences. Just an 💡  we could add more instructions needed, like how to setup multiple projects if only the web API is running.